cmd/proxy/cmd/proxy.go

```diff
@@ -321,6 +328,8 @@ func (c *ClusterChecker) TimeoutChecker(checkOkCh chan struct{}) {
 			timeoutTimer.Stop()
 
 			c.configMutex.Lock()
+			reqTimeout, _ := c.e.(*store.KVBackedStore).GetRequestTimeout()
```

Comment:
Can we add new methods to interface to avoid explicit cast? Perhaps, adding stubs for other implementations. If we don't check for error, is there sense in returning it? Perhaps, panic() is fine if error is unlikely.

Reply:
refactored, see new edition